agent_server.py

```
@@ -284,6 +304,47 @@ def _collect_agent_status_snapshot() -> Dict[str, Any]:
+    """
+    if not isinstance(messages, list):
+        return None
+    user_parts: list[str] = []
+    for m in messages:
+        if not isinstance(m, dict):
+            continue
+        if m.get("role") != "user":
+            continue
+        # Keep text as primary signal, and attach optional metadata if present.
+        if "text" in m and "content" not in m:
+            logger.warning(f"[AgentServer] Deprecation Warning: Received user message with 'text' but no 'content'.")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)